src/psm/src/ir_solver.cpp

```diff
@@ -411,6 +411,13 @@ bool IRSolver::CreateJ() {  // take current_map as an input?
       continue;
     }
     int x, y;
+    if (!inst-> getPlacementStatus().isPlaced()) {
+      m_logger->warn(utl::PSM, 71, "Instance {} is not placed. The location "
+                     "of its current source is being moved to (0,0). The IR " 
+                     "drop values will be incorrect. Please run "
+                     "analyze_power_grid after all instances are placed. ",
+                     it->first);
+    }
     inst->getLocation(x, y);
```

Comment:
There is no point to get the location of an unplaced instance.  You should just skip them after warning with a continue.